Makefile

```diff
@@ -0,0 +1,44 @@
+.PHONY: auth clean gem publish test
+
+NAME?=json_rails_logger
+OWNER?=epimorphics
+VERSION?=$(shell ruby -e 'require "./lib/json_rails_logger/version" ; puts JsonRailsLogger::VERSION')
+PAT?=$(shell read -p 'Github access token:' TOKEN; echo $$TOKEN)
+
+AUTH=${HOME}/.gem/credentials
+GEM=${NAME}-${VERSION}.gem
+GPR=https://rubygems.pkg.github.com/${OWNER}
+SPEC=${NAME}.gemspec
+
+all: publish
+
+${AUTH}:
+	@mkdir -p ${HOME}/.gem
+	@echo '---' > $@
```

Comment:
I know it's part of make syntax, but I think it would be a lot clearer to the casual reader if `$@` was replaced by `${AUTH}`